src/System.Linq.Queryable/tests/System.Linq.Queryable.Tests.csproj

```diff
@@ -70,6 +72,11 @@
       <Name>System.Linq.Queryable</Name>
       <Private>true</Private>
     </ProjectReference>
+    <ProjectReference Include="..\..\System.Linq\src\System.Linq.csproj">
```

Comment:
Highlighting this bit of mine, because I'm not sure I did the right thing with how this reference is in here, and if I didn't I don't want it to go unnoticed.